packages/loader/src/gltf/schema.ts

```diff
@@ -623,71 +595,63 @@ export interface GlTf {
   /**
    * An array of accessors.
    */
-  accessors?: Accessor[];
+  accessors?: IAccessor[];
   /**
    * An array of keyframe animations.
    */
-  animations?: Animation[];
+  animations?: IAnimation[];
   /**
    * Metadata about the glTF asset.
    */
-  asset: Asset;
+  asset: IAsset;
   /**
    * An array of buffers.
    */
-  buffers?: Buffer[];
+  buffers?: IBuffer[];
   /**
    * An array of bufferViews.
    */
-  bufferViews?: BufferView[];
+  bufferViews?: IBufferView[];
   /**
    * An array of cameras.
    */
-  cameras?: Camera[];
+  cameras?: ICamera[];
   /**
    * An array of images.
    */
-  images?: Image[];
+  images?: IImage[];
   /**
    * An array of materials.
    */
-  materials?: Material[];
+  materials?: IMaterial[];
   /**
    * An array of meshes.
    */
-  meshes?: Mesh[];
+  meshes?: IMesh[];
   /**
    * An array of nodes.
    */
-  nodes?: Node[];
+  nodes?: INode[];
   /**
    * An array of samplers.
    */
-  samplers?: Sampler[];
+  samplers?: ISampler[];
   /**
    * The index of the default scene.
    */
-  scene?: GlTfId;
+  scene?: GLTFID;
   /**
    * An array of scenes.
    */
-  scenes?: Scene[];
+  scenes?: IScene[];
   /**
    * An array of skins.
    */
-  skins?: Skin[];
+  skins?: ISkin[];
   /**
    * An array of textures.
    */
-  textures?: Texture[];
+  textures?: ITexture[];
   extensions?: any;
   extras?: any;
-  [k: string]: any;
-}
-
-export interface LoadedGLTFResource {
-  buffers?: ArrayBuffer[];
-  textures?: Texture2D[];
-  shaders?: string[];
-  gltf?: GlTf;
 }
```

Comment:
1.The file name should `Schema.ts`, not `schema.ts`
2. You can write the interfaces with specification: https://github.com/KhronosGroup/glTF/tree/master/specification/2.0/schema , for example use `extension` to write the file same as official json.

Reply:
done